### ISSUE TITLE
Assignment 2 Comments

### ISSUE THREAD

**Issue by tilaks26** [airavata-courses/hiteshkumardasika#6]
I have a few suggestions/comments -

- Can you please update your architecture diagram with the components and flow of control?
- Ideally, you should not link the Docker containers to RabbitMQ. Access the RabbitMQ using it's public IP from the Docker containers (microservices).

Can you please make these changes and link this issue with the commit so I can review it?

**Comment by tilaks26** [comment]
- Can I please have your Jenkins credentials.
- Also, what exactly does your app do? I tried testing it. When I ender a todo item, it replaces the one that was present before. Also, the delete button is not working.

**Comment by hiteshkumardasika** [comment]
@tilaks26 
This is my Jenkins URL 129.114.104.36:8080
Username:password == root:root

I used the blueocean plugin to generate my pipeline. It is in my jenkins in the left menu saying "Open Blue Ocean".

My app just makes a todolist. When you add a todo item, it is displayed in the below textbox. Im not sure why delete is not working because it is getting deleted in the db. I will check that and will fix it.

**Comment by tilaks26** [comment]
Please fix the application. The communication between the services through the RabbitMQ might be broken.

**Comment by marpierc** [comment]
@hiteshkumardasika You'll need to resubmit your assignment with the identified corrections.

**Comment by hiteshkumardasika** [comment]
@tilaks26 
The calls are working fine. When you give an Id to delete, the item is getting deleted but the request is not reflecting that and im suspecting Rabbitmq is caching the request. I tried adding the 'Lazy Queue' argument as shown by the rabbitmq (https://www.rabbitmq.com/lazy-queues.html), but it didn't reflect the result.

But when i add a new item, the deleted item will not be shown in the Item list. I will try some other options. Any suggestions are welcome. 
P.S. The request is not being cached on the Ajax side because I disabled the cache.
